doc/doc/basics/extractors.md

```diff
@@ -17,10 +17,15 @@ class contains different extractor *styles*:
 
 - Row-wise extractors:
   - [`json_extractor`](#json_extractor): highly flexible and compatible with
-  previous systems, but with limited performance 
+  previous systems, but with limited performance
```

Comment:
I see. Sorry about the trouble... I'll rebase after the release and handle the conflicts from my side so that this change stays automatically mergeable.

If the conflicts you mentioned were b/w master and develop, I think the real solution is to have one conceptual "master" branch and avoid changes flying in both directions (colliding with each other).